redbot/cogs/mod/kickban.py

```diff
@@ -292,6 +292,7 @@ async def hackban(
         using this command"""
         banned = []
         errors = {}
+        upgrades = {}
```

Comment:
You can just use a list and append IDs to it instead of using a dict with two same values.

Reply:
yes... that would've been smart